proto/gds/members/v1alpha1/members.proto

```diff
@@ -62,6 +62,9 @@ message VASPMember {
 message SummaryRequest {
     // The start date for determining how many members are new - optional
     string since = 1;
+
+    // The VASP ID to retrieve details for - optional
+    string vasp = 2;
```

Comment:
Should we change this to `member_id` to match your other changes? If not, can we at least make it `vasp_id`?

```suggestion
    // Include your VASP ID for details about your VASP record included with the summary - optional
    string vasp_id = 2;
```